src/footer-components/FooterSection.tsx

```diff
@@ -1,6 +1,16 @@
+interface Link {
+  url: string;
+  name: React.ReactNode;
+}
 
+interface FooterSectionProps {
+  title: string;
+  links: Link[];
+  isHorizontal: boolean;
+}
 
```

Comment:
These interfaces could be added to the types file too

Reply:
done it